lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/SamigoExport.java

```diff
@@ -233,6 +233,8 @@ private void outputQuestions(CCConfig ccConfig, List<ItemDataIfc> itemList, Stri
                     .sorted(Comparator.comparing(AnswerIfc::getSequence))
                     .collect(Collectors.toList());
 
+            Double score = answers.get(0).getScore();
```

Comment:
can add a tiny bit of protection here in case answers is empty ?

Reply:
Sure!, but an instructor can't save a question without at least one answer.

Reply:
Agree..... but we see some very odd NPEs occasionally .....

Reply:
Done! @ottenhoff